core/core.mk

```diff
@@ -99,6 +99,27 @@ libname = utils
 libdir = lib/libutils
 include mk/lib.mk
 
+ifeq ($(CFG_CORE_SE05X), y)
+CFG_CORE_SE05X_SCP03_PROVISION ?= n
+CFG_CORE_SE05X_DISPLAY_INFO ?= y
+CFG_CORE_SE05X_SCP03_EARLY ?= y
+CFG_CORE_SE05X_INIT_NVM ?= n
+CFG_CORE_SE05X_OEFID ?= n
+# Extra stacks required to support the Plug and Trust external library
+CFG_STACK_THREAD_EXTRA ?= 8192
+CFG_STACK_TMP_EXTRA ?= 8192
+$(call force, CFG_CRYPTO_RSASSA_NA1, n, not supported by se050)
+$(call force, CFG_NXP_SE05X_SVC, y)
+$(call force, CFG_NXP_SE05X_RNG_DRV, y)
+# plug and trust include files
+core-platform-cflags += "-I${CFG_NXP_SE05X_EXT_REPO}/optee_lib/include"
+libdeps-external := "${CFG_NXP_SE05X_EXT_LIB}"
```

Comment:
Could this block up to this point be moved to `core/lib/libnxpse050/conf.mk`? `mk/lib.mk` would take care of including such a conf file if it exists.